multiclient.go

```diff
@@ -375,6 +375,11 @@ func (m *MultiClient) sendWithClient(clientID int, dests []string, payload *payl
 	if client == nil {
 		return ErrNilClient
 	}
+
+	dests, err := ResolveDests(dests, m.config.Resolver)
```

Comment:
I think it's better to put resolve logic in these 3 functions:
```
client.Send
multiclient.Send
multiclient.SendWithClient
```
so internal send calls (e.g. reply) won't call resolver